app/generators/new_page_generator.rb

```diff
@@ -1,17 +1,19 @@
 class NewPageGenerator
   include ActiveModel::Model
+  include ApplicationHelper
```

Comment:
Is the `strip_url` the reason of the included module?

Reply:
yeah unfortunately so